sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/statsbeat/_statsbeat.py

```diff
@@ -0,0 +1,108 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+import os
+import threading
+
+from opentelemetry.sdk.metrics import MeterProvider
+from opentelemetry.sdk.metrics.export import PeriodicExportingMetricReader
+
+from azure.monitor.opentelemetry.exporter.statsbeat._exporter import _StatsBeatExporter
+from azure.monitor.opentelemetry.exporter.statsbeat._statsbeat_metrics import _StatsbeatMetrics
+
+# pylint: disable=line-too-long
+_DEFAULT_NON_EU_STATS_CONNECTION_STRING = "InstrumentationKey=c4a29126-a7cb-47e5-b348-11414998b11e;IngestionEndpoint=https://westus-0.in.applicationinsights.azure.com/"
+_DEFAULT_EU_STATS_CONNECTION_STRING = "InstrumentationKey=7dc56bab-3c0c-4e9f-9ebb-d1acadee8d0f;IngestionEndpoint=https://westeurope-5.in.applicationinsights.azure.com/"
+_DEFAULT_STATS_SHORT_EXPORT_INTERVAL = 900  # 15 minutes
+_DEFAULT_STATS_LONG_EXPORT_INTERVAL = 86400  # 24 hours
+_EU_ENDPOINTS = [
+    "westeurope",
+    "northeurope",
+    "francecentral",
+    "francesouth",
+    "germanywestcentral",
+    "norwayeast",
+    "norwaywest",
+    "swedencentral",
+    "switzerlandnorth",
+    "switzerlandwest",
+]
+
+_STATSBEAT_METER_PROVIDER = None
+_STATSBEAT_LOCK = threading.Lock()
+
+# pylint: disable=global-statement
+# pylint: disable=protected-access
+def collect_statsbeat_metrics(exporter) -> None:
+    global _STATSBEAT_METER_PROVIDER
+    # Only start statsbeat if did not exist before
+    if _STATSBEAT_METER_PROVIDER is None:
+        with _STATSBEAT_LOCK:
+            statsbeat_exporter = _StatsBeatExporter(
+                connection_string=_get_stats_connection_string(exporter._endpoint),
+            )
+            reader = PeriodicExportingMetricReader(
+                statsbeat_exporter,
+                export_interval_millis=_get_stats_short_export_interval() * 1000,  # 15m by default
+            )
+            _STATSBEAT_METER_PROVIDER = MeterProvider(metric_readers=[reader])
+            statsbeat_metrics = _StatsbeatMetrics(
+                _STATSBEAT_METER_PROVIDER,
+                exporter._instrumentation_key,
+                exporter._endpoint,
+            )
+            return statsbeat_metrics
```

Comment:
This return value does not seem used and the method header is -> None. What is this needed for?

Reply:
Changed to return nothing.